packages/components/src/border-box-control/stories/index.js

```diff
@@ -84,12 +84,12 @@ Default.args = {
 		style: 'dashed',
 		width: '1px',
 	},
-	__next36pxDefaultSize: false,
+	size: 'default',
```

Comment:
We should be looking into migrating `BorderControl` and `BorderBoxControl` stories to TypeScript and controls, since at the moment there's no easy way to set the size to `'__unstable-large'` without typing it

Reply:
@mirka is already one step ahead and has already converted the `BorderControl` stories to TypeScript and Controls a couple of months ago. I'll get a PR together to bring `BorderBoxControl` up-to-date as a follow-up to this.

Reply:
A quick refactor of the `BorderBoxControl` stories to TypeScript etc is up: https://github.com/WordPress/gutenberg/pull/45002.